indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -201,7 +201,14 @@ def _check_access(self):
             raise Forbidden
 
     def _process(self):
+        service_url = editing_settings.get(self.event, 'service_url')
```

Comment:
I'd just do `if editing_settings.get(self.event, 'service_url'):` below.. getting that setting will work regardless of whether the editable exist or not